trello_items.py

```diff
@@ -0,0 +1,94 @@
+from flask import session
+import requests
+import config
+import os
+from item import Item
+
+key = config.key
+token = config.token
+
+def get_items():
+    """
+    Fetches all cards from Trello.
+
+    Returns:
+        list: The list of cards constructed using the Card class.
+    """
+    url1 = "https://api.trello.com/1/members/me/boards"
+    query = {
+        'key': key,
+        'token': token
+    }
+    boards = requests.request("GET", url1, params=query)
+    all_cards = []
+    for board in boards.json():
+        board_id = board['id']
+        url2 = "https://api.trello.com/1/boards/{}/cards".format(board_id)
```

Comment:
If you have a recent version of python you should be able to write this like `f"https://api.trello.com/1/boards/{board_id}/cards"`